src/components/Error/ErrorFallback.tsx

```diff
@@ -14,6 +14,7 @@ const ErrorFallback: FC<Props> = ({ error }) => {
 
   if (error instanceof ZoomError) {
     /** ToDo: 401: {"code":124,"message":"Access token is expired."} */
+    /** ToDo: 404: {"code":3001,"message":"Конференция не найдена, или срок ее действия истек."} */
```

Comment:
Try and keep it english please

Reply:
oh sorry, it's just a note about the error format))) **it takes the language from the profile settings**

Reply:
I will do more detailed error handling in this story https://app.shortcut.com/deskpro/story/106868/zoom-improvements

Reply:
If ash approves this alright, because we're now required to make tests in order to merge branches